doc/Linux_deployment.md

```diff
@@ -0,0 +1,24 @@
+# Deploying portable **XMR-Stak** on Linux systems
+
+XMR-Stak releases include a pre-built portable version. If you are simply using it to avoid having to compile the application, you can simply download **xmr-stak-portable-bin.tar.gz** from our [latest releases](https://github.com/fireice-uk/xmr-stak/releases/latest). Open up command line, and use the following commands:
+
+```
+tar xzf xmr-stak-portable-bin.tar.gz
```

Comment:
Can we rename it to `xmr-stak-portable-linux` here and on all other places? To avoid that windows user download it.

Reply:
Good idea.